examples/createContract/src/spec.md

```diff
@@ -23,3 +53,31 @@ iff
     VCallValue == 0
     VCallDepth <Int 1024
 ```
+
+### create2
+
+```act
+behaviour create_kid1 of Mom
+interface create_kid1()
+
+types
+
+    KID    : address Kid
+
+creates storage KID
+
+    #Kid.parent |-> ACCT_ID
+
+storage
+
+    #Mom.kid2 |-> 0 => KID
+
+if
+
+    #newAddr(ACCT_ID, 1, Mom_bin_runtime) == KID
```

Comment:
We want the contract `initcode` here rather than the runtime - assume this will need adding to the out spec?